code/modules/mob/living/carbon/superior_animal/defense.dm

```diff
@@ -153,21 +153,27 @@
 			blinded = TRUE
 			stat = UNCONSCIOUS
 			if(halloss > 0)
-				adjustHalLoss(-3)
+				adjustHalLoss(-4)
 
 		if(sleeping)
-			adjustHalLoss(-3)
+			adjustHalLoss(-5)
+			if(bruteloss > 0)
+				adjustBruteLoss(-0.25)
+			if(toxloss > 0)
+				adjustToxLoss(-0.25)
+			if(fireloss > 0)
+				adjustFireLoss(-0.25)	
 			sleeping = max(sleeping-1, 0)
 			blinded = TRUE
 			stat = UNCONSCIOUS
 		else if(resting)
 			if(halloss > 0)
-				adjustHalLoss(-3)
+				adjustHalLoss(-4)
 
 		else
 			stat = CONSCIOUS
 			if(halloss > 0)
-				adjustHalLoss(-1)
+				adjustHalLoss(-2)
```

Comment:
pretty sure this doubles pain regen when not sleeping

Reply:
Intended to mitigate the impact and prolific pain applications that would be harder to hunt down than just slapping this on. 

Would you suggest increasing it to 1.5 instead?